JotunnLib/Utils/SimpleJVL.cs

```diff
@@ -0,0 +1,272 @@
+                Craft.m_repairItemDoneEffects = craft.m_repairItemDoneEffects;
+                Craft.m_roofCheckPoint = craft.m_roofCheckPoint;
+                Craft.m_showBasicRecipies = craft.m_showBasicRecipies;
+                Craft.m_updateExtensionTimer = craft.m_updateExtensionTimer;
+                Craft.m_useAnimation = craft.m_useAnimation;
+                Craft.m_useDistance = craft.m_useDistance;
+                Craft.m_useTimer = craft.m_useTimer;
+            }
+            else
+            {
+                reflect.m_name = name;
+            }
+        }
+
+        /// <summary>
+        ///     Adds a <see cref="CraftingStation"/> to ObjectDB with inherited properties by default, and builds a new recipe for it.
+        /// </summary>
+        /// <param name="prefabName">Case sensitive name of the new Piece.</param>
+        /// <param name="name">Token name of the new item.</param>
+        /// <param name="description">Description of the item, shows up in tooltip.</param>
+        /// <param name="pieceTable">Case sensitive name of PieceTable assigned to the recipe.</param>
+        /// <param name="craftingStation">Case sensitive name of the CraftingStation required </param>
+        /// <param name="inputs">Recipe config as a <see cref="RequirementConfig"/></param>
+        public void AddStation(string prefabName, string name, string description, GameObject prefab, string pieceTable,
+            string craftingStation, bool isAllowedInDungeons, params RequirementConfig[] inputs)
+        {
+            AddStationPiece(prefabName, name, description);
+            AddPieceRequirements(prefab, pieceTable, craftingStation, isAllowedInDungeons, inputs);
+        }
+
+        /// <summary>
+        ///     Adds the appropriate <see cref="ConversionConfig"/> based on which <see cref="CraftingStation"/> is used.
+        /// </summary>
+        /// <param name="station">Case sensitive name of the <see cref="CraftingStation"/> the recipe is made at.</param>
+        /// <param name="fromitem">Case sensitive name of item to be converted.</param>
+        /// <param name="toitem">Case sensitive name of item to be produced.</param>
+        public void AddConversion(string station, string fromitem, string toitem)
+        {
+            CustomItemConversion conversion;
+            switch (station)
+            {
+                case "piece_cookingstation":
+                    conversion = new CustomItemConversion(new CookingConversionConfig {Station = station, FromItem = fromitem, ToItem = toitem});
+                    ItemManager.Instance.AddItemConversion(conversion);
+                    break;
+                case "fermenter":
+                    conversion = new CustomItemConversion(new FermenterConversionConfig {Station = station, FromItem = fromitem, ToItem = toitem});
+                    ItemManager.Instance.AddItemConversion(conversion);
+                    break;
+                case "smelter":
```

Comment:
Lacks 60% of the conversions, does not work with custom conversion pieces (magic strings)